orderbook/src/fee.rs

```diff
@@ -226,24 +234,30 @@ impl MinFeeCalculator {
         };
         let estimate = self.price_estimator.estimate(&query).await?;
         let price = estimate.price_in_sell_token_f64(&query);
+
+        tracing::debug!(
+            "computed unsubsidized fee amount of {} ETH at a price of {} ETH/{:?}",
+            fee_in_eth,
+            price,
+            sell_token,
+        );
```

Comment:
Not sure which is the right way to go here, but this makes sense to me reading as "ETH per SELL_TOKEN". I think Rafa might say its the other way around because it makes sense to me and I am always wrong about this. What do we expect this to say if `sell_token = DAI` or `USDC`?